### ISSUE TITLE
openjdk11_j9_extended.functional_x86-64_linux_xl SyntheticGCWorkload_DoubleMap_J9_0 Segmentation error vmState=0x00020004

### ISSUE THREAD

**Issue by JasonFengJ9** [eclipse/openj9#8345]
Failure link
------------

https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_x86-64_linux_xl_Nightly/250/tapResults/

Optional info
-------------

Failure output (captured from console output)
---------------------------------------------

```
Unhandled exception
Type=Segmentation error vmState=0x00020004
J9Generic_Signal_Number=00000018 Signal_Number=0000000b Error_Value=00000000 Signal_Code=00000001
Handler1=00007FA5E1082870 Handler2=00007FA5E0970090 InaccessibleAddress=00007FA533F03278
RDI=00007FA5DC00F8C0 RSI=00007FA531DF7438 RAX=00007FA533F03260 RBX=00007FA531DF7438
RCX=00007FA531178580 RDX=0000000000000190 R8=00007FA531178588 R9=00007FA531DF7438
R10=00007FA5DAB7B770 R11=0000000000000020 R12=0000000000000000 R13=0000000000000190
R14=0000000004600002 R15=00007FA531DF7554
Unhandled exception
Type=Segmentation error vmState=0x00020004
J9Generic_Signal_Number=00000018 Signal_Number=0000000b Error_Value=00000000 Signal_Code=00000001
Handler1=00007FA5E1082870 Handler2=00007FA5E0970090 InaccessibleAddress=00007FA533F03278
RIP=00007FA5DB2B92C3 GS=0000 FS=0000 RSP=00007FA5311784F0
EFlags=0000000000010246 CS=0033 RBP=000000000000018F ERR=0000000000000004
TRAPNO=000000000000000E OLDMASK=0000000000000000 CR2=00007FA533F03278
RDI=00007FA5DC00F8C0 RSI=00007FA531DF7438 RAX=00007FA533F03260 RBX=00007FA531DF7438
RCX=00007FA5311B9580 RDX=0000000000000190 R8=00007FA5311B9588 R9=00007FA531DF7438
R10=00007FA5DAB7B770 R11=0000000000000001 R12=0000000000000000 R13=0000000000000190
R14=0000000004600002 R15=00007FA531DF7554
RIP=00007FA5DB2B92C3 GS=0000 FS=0000 RSP=00007FA5311B94F0
EFlags=0000000000010246 CS=0033 RBP=000000000000018F ERR=0000000000000004
TRAPNO=000000000000000E OLDMASK=0000000000000000 CR2=00007FA533F03278
xmm0 0000000300000003 (f: 3.000000, d: 6.365987e-314)
xmm1 0000000200000003 (f: 3.000000, d: 4.243992e-314)
xmm2 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm3 00007fa5d9fff340 (f: 3657429760.000000, d: 6.934226e-310)
xmm4 00007fa5d9fff4b0 (f: 3657430272.000000, d: 6.934226e-310)
xmm5 00007fa5d9fff620 (f: 3657430528.000000, d: 6.934226e-310)
xmm6 00007fa5d9fff790 (f: 3657431040.000000, d: 6.934226e-310)
xmm7 00007fa5d9fff900 (f: 3657431296.000000, d: 6.934226e-310)
xmm8 00007fa5d9fffa70 (f: 3657431552.000000, d: 6.934226e-310)
xmm9 00007fa5d9fffbe0 (f: 3657432064.000000, d: 6.934226e-310)
xmm10 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm11 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm12 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm13 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm15 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm0 00007fa5311b9aa0 (f: 823892608.000000, d: 6.934086e-310)
xmm1 00750064006f006d (f: 7274605.000000, d: 1.869198e-306)
xmm2 006d006f0068002f (f: 6815791.000000, d: 1.290618e-306)
xmm3 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm4 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm5 0074007300650054 (f: 6619220.000000, d: 1.780215e-306)
xmm6 006b0064006a006e (f: 6946926.000000, d: 1.201608e-306)
xmm7 00780065005f0039 (f: 6225977.000000, d: 2.136208e-306)
xmm8 0066002e00640065 (f: 6553701.000000, d: 9.790637e-307)
xmm9 0061006e006f0069 (f: 7274601.000000, d: 7.565998e-307)
xmm10 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm11 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm12 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm13 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm15 0000000000000000 (f: 0.000000, d: 0.000000e+00)
Module=/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_x86-64_linux_xl_Nightly/openjdkbinary/j2sdk-image/lib/default/libj9jit29.so
Module_base_address=00007FA5DB013000
Module=/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_x86-64_linux_xl_Nightly/openjdkbinary/j2sdk-image/lib/default/libj9jit29.so
Module_base_address=00007FA5DB013000
Target=2_90_20200117_265 (Linux 4.4.0-169-generic)
CPU=amd64 (4 logical CPUs) (0x5e2f07000 RAM)
Target=2_90_20200117_265 (Linux 4.4.0-169-generic)
CPU=amd64 (4 logical CPUs) (0x5e2f07000 RAM)

JavaVMInitArgs.nOptions=19:
    -Xoptionsfile=/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_x86-64_linux_xl_Nightly/openjdkbinary/j2sdk-image/lib/options.default
    -Xlockword:mode=default,noLockword=java/lang/String,noLockword=java/util/MapEntry,noLockword=java/util/HashMap$Entry,noLockword=org/apache/harmony/luni/util/ModifiedMap$Entry,noLockword=java/util/Hashtable$Entry,noLockword=java/lang/invoke/MethodType,noLockword=java/lang/invoke/MethodHandle,noLockword=java/lang/invoke/CollectHandle,noLockword=java/lang/invoke/ConstructorHandle,noLockword=java/lang/invoke/ConvertHandle,noLockword=java/lang/invoke/ArgumentConversionHandle,noLockword=java/lang/invoke/AsTypeHandle,noLockword=java/lang/invoke/ExplicitCastHandle,noLockword=java/lang/invoke/FilterReturnHandle,noLockword=java/lang/invoke/DirectHandle,noLockword=java/lang/invoke/ReceiverBoundHandle,noLockword=java/lang/invoke/DynamicInvokerHandle,noLockword=java/lang/invoke/FieldHandle,noLockword=java/lang/invoke/FieldGetterHandle,noLockword=java/lang/invoke/FieldSetterHandle,noLockword=java/lang/invoke/StaticFieldGetterHandle,noLockword=java/lang/invoke/StaticFieldSetterHandle,noLockword=java/lang/invoke/IndirectHandle,noLockword=java/lang/invoke/InterfaceHandle,noLockword=java/lang/invoke/VirtualHandle,noLockword=java/lang/invoke/PrimitiveHandle,noLockword=java/lang/invoke/InvokeExactHandle,noLockword=java/lang/invoke/InvokeGenericHandle,noLockword=java/lang/invoke/VarargsCollectorHandle,noLockword=java/lang/invoke/ThunkTuple
    -Xjcl:jclse29
    -Dcom.ibm.oti.vm.bootstrap.library.path=/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_x86-64_linux_xl_Nightly/openjdkbinary/j2sdk-image/lib/default:/home/jenkins/workspace/Test_openjdk11_j9_extended.functiona
JavaVMInitArgs.nOptions=19:
    -Xoptionsfile=/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_x86-64_linux_xl_Nightly/openjdkbinary/j2sdk-image/lib/options.default
    -Xlockword:mode=default,noLockword=java/lang/String,noLockword=java/util/MapEntry,noLockword=java/util/HashMap$Entry,noLockword=org/apache/harmony/luni/util/ModifiedMap$Entry,noLockword=java/util/Hashtable$Entry,noLockword=java/lang/invoke/MethodType,noLockword=java/lang/invoke/MethodHandle,noLockword=java/lang/invoke/CollectHandle,noLockword=java/lang/invoke/ConstructorHandle,noLockword=java/lang/invoke/ConvertHandle,noLockword=java/lang/invoke/ArgumentConversionHandle,noLockword=java/lang/invoke/AsTypeHandle,noLockword=java/lang/invoke/ExplicitCastHandle,noLockword=java/lang/invoke/FilterReturnHandle,noLockword=java/lang/invoke/DirectHandle,noLockword=java/lang/invoke/ReceiverBoundHandle,noLockword=java/lang/invoke/DynamicInvokerHandle,noLockword=java/lang/invoke/FieldHandle,noLockword=java/lang/invoke/FieldGetterHandle,noLockword=java/lang/invoke/FieldSetterHandle,noLockword=java/lang/invoke/StaticFieldGetterHandle,noLockword=java/lang/invoke/StaticFieldSetterHandle,noLockword=java/lang/invoke/IndirectHandle,noLockword=java/lang/invoke/InterfaceHandle,noLockword=java/lang/invoke/VirtualHandle,noLockword=java/lang/invoke/PrimitiveHandle,noLockword=java/lang/invoke/InvokeExactHandle,noLockword=java/lang/invoke/InvokeGenericHandle,noLockword=java/lang/invoke/VarargsCollectorHandle,noLockword=java/lang/invoke/ThunkTuple
    -Xjcl:jclse29
    -Dcom.ibm.oti.vm.bootstrap.library.path=/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_x86-64_linux_xl_Nightly/openjdkbinary/j2sdk-image/lib/default:/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_x86-64_linuxUnhandled exception
Type=Segmentation error vmState=0x00020004
J9Generic_Signal_Number=00000018 Signal_Number=0000000b Error_Value=00000000 Signal_Code=00000001
Handler1=00007FA5E1082870 Handler2=00007FA5E0970090 InaccessibleAddress=00007FA533F03278
RDI=00007FA5DC00F8C0 RSI=00007FA531DF7438 RAX=00007FA533F03260 RBX=00007FA531DF7438
RCX=00007FA531137580 RDX=0000000000000190 R8=00007FA531137588 R9=00007FA531DF7438
R10=00007FA5DAB7B770 R11=000000000000002D R12=0000000000000000 R13=0000000000000190
R14=0000000004600002 R15=00007FA531DF7554
RIP=00007FA5DB2B92C3 GS=0000 FS=0000 RSP=00007FA5311374F0
EFlags=0000000000010246 CS=0033 RBP=000000000000018F ERR=0000000000000004
TRAPNO=000000000000000E OLDMASK=0000000000000000 CR2=00007FA533F03278
xmm0 00007fa531137aa0 (f: 823360128.000000, d: 6.934086e-310)
xmm1 0000000100000000 (f: 0.000000, d: 2.121996e-314)
xmm2 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm3 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm4 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm5 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm6 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm7 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm8 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm9 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm10 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm11 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm12 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm13 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm15 0000000000000000 (f: 0.000000, d: 0.000000e+00)
Module=/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_x86-64_linux_xl_Nightly/openjdkbinary/j2sdk-image/lib/default/libj9jit29.so
Module_base_address=00007FA5DB013000
Target=2_90_20200117_265 (Linux 4.4.0-169-generic)
CPU=amd64 (4 logical CPUs) (0x5e2f07000 RAM)

JavaVMInitArgs.nOptions=19:
    -Xoptionsfile=/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_x86-64_linux_xl_Nightly/openjdkbinary/j2sdk-image/lib/options.default
    -Xlockword:mode=default,noLockword=java/lang/String,noLockword=java/util/MapEntry,noLockword=java/util/HashMap$Entry,noLockword=org/apache/harmony/luni/util/ModifiedMap$Entry,noLockword=java/util/Hashtable$Entry,noLockword=java/lang/invoke/MethodType,noLockword=java/lang/invoke/MethodHandle,noLockword=java/lang/invoke/CollectHandle,noLockword=java/lang/invoke/ConstructorHandle,noLockword=java/lang/invoke/ConvertHandle,noLockword=java/lang/invoke/ArgumentConversionHandle,noLockword=java/lang/invoke/AsTypeHandle,noLockword=java/lang/invoke/ExplicitCastHandle,noLockword=java/lang/invoke/FilterReturnHandle,noLockword=java/lang/invoke/DirectHandle,noLockword=java/lang/invoke/ReceiverBoundHandle,noLockword=java/lang/invoke/DynamicInvokerHandle,noLockword=java/lang/invoke/FieldHandle,noLockword=java/lang/invoke/FieldGetterHandle,noLockword=java/lang/invoke/FieldSetterHandle,noLockword=java/lang/invoke/StaticFieldGetterHandle,noLockword=java/lang/invoke/StaticFieldSetterHandle,noLockword=java/lang/invoke/IndirectHandle,noLockword=java/lang/invoke/InterfaceHandle,noLockword=java/lang/invoke/VirtualHandle,noLockword=java/lang/invoke/PrimitiveHandle,noLockword=java/lang/invoke/InvokeExactHandle,noLockword=java/lang/invoke/InvokeGenericHandle,noLockword=java/lang/invoke/VarargsCollectorHandle,noLockword=java/lang/invoke/ThunkTuple
    -Xjcl:jclse29
    -Dcom.ibm.oti.vm.bootstrap.library.path=/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_x86-64_linux_xl_Nightly/openjdkbinary/j2sdk-image/lib/default:/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_x86-64_linux_xl_Nightly/openjdkbinary/j2sdk-image/lib
    -DJVMPORT030W /proc/sys/kernel/core_pattern setting "|/usr/share/apport/apport %p %s %c %d %P" specifies that the core dump is to be piped to an external program.  Attempting to rename either core or core.12415.

JVMPORT030W /proc/sys/kernel/core_pattern setting "|/usr/share/apport/apport %p %s %c %d %P" specifies that the core dump is to be piped to an external program.  Attempting to rename either core or core.12416.


Generated system dump: core
----------- Stack Backtrace -----------
(0x00007FA5DB2B92C3 [libj9jit29.so+0x2a62c3])
(0x00007FA5DB2B9CF1 [libj9jit29.so+0x2a6cf1])
(0x00007FA5DB97BF47 [libj9jit29.so+0x968f47])
(0x00007FA5DB97CC8C [libj9jit29.so+0x969c8c])
(0x00007FA5E10B20FE [libj9vm29.so+0xc60fe])
(0x00007FA5DAB7B836 [libj9gc29.so+0x3f836])
(0x00007FA5DAB7006D [libj9gc29.so+0x3406d])
(0x00007FA5DAB6EDAF [libj9gc29.so+0x32daf])
(0x00007FA5DAC067F9 [libj9gc29.so+0xca7f9])
(0x00007FA5DAC07B5D [libj9gc29.so+0xcbb5d])
(0x00007FA5DAC6B53B [libj9gc29.so+0x12f53b])
(0x00007FA5DAC6B031 [libj9gc29.so+0x12f031])
(0x00007FA5E0970DF3 [libj9prt29.so+0x20df3])
(0x00007FA5DAC6ABBC [libj9gc29.so+0x12ebbc])
(0x00007FA5E0DDF1A6 [libj9thr29.so+0xe1a6])
(0x00007FA5E2CC16BA [libpthread.so.0+0x76ba])
clone+0x6d (0x00007FA5E25E241D [libc.so.6+0x10741d])
---------------------------------------

SyntheticGCWorkload_DoubleMap_J9_0_FAILED
```

**Comment by JasonFengJ9** [comment]
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_s390x_linux_Nightly/262/tapResults/

```
Unhandled exception
Type=Segmentation error vmState=0x00000000
J9Generic_Signal_Number=00000018 Signal_Number=0000000b Error_Value=3011d068 Signal_Code=00000001
Handler1=000003FFA0F44558 Handler2=000003FFA0B28098 InaccessibleAddress=000003FF4017E000
gpr0=000003FF9BFB2E98 gpr1=000003FF4017E220 gpr2=000003FF4017E1B0 gpr3=000003FF9C091500
gpr4=000003FF9B492DB6 gpr5=0000000000000001 gpr6=00059C6538EEA0EA gpr7=000003FFA1143F08
gpr8=000003FF000075A0 gpr9=000003FF9C08B608 gpr10=000003FF9C091500 gpr11=0000000000000068
gpr12=000003FFA199B000 gpr13=000003FF9BD76D68 gpr14=000003FF9B492DCC gpr15=000003FF951FBCA8
psw=000003FF9B492DD4 mask=0705200180000000 fpc=00080000 bea=000003FF9B48FA7E
fpr0 000003ffa199b000 (f: 2711203840.000000, d: 2.172141e-311)
fpr1 000003ffa197c84f (f: 2711078912.000000, d: 2.172141e-311)
fpr2 0000000000000001 (f: 1.000000, d: 4.940656e-324)
fpr3 000003ffa1a01078 (f: 2711621632.000000, d: 2.172141e-311)
fpr4 000003ff912b85f0 (f: 2435548672.000000, d: 2.172005e-311)
fpr5 000003ffa197b818 (f: 2711074816.000000, d: 2.172141e-311)
fpr6 0000000000000000 (f: 0.000000, d: 0.000000e+00)
fpr7 bc69000000000000 (f: 0.000000, d: -1.084202e-17)
fpr8 0000000000040000 (f: 262144.000000, d: 1.295163e-318)
fpr9 0000000000000000 (f: 0.000000, d: 0.000000e+00)
fpr10 000003ff951bf000 (f: 2501636096.000000, d: 2.172038e-311)
fpr11 000003ffce87f253 (f: 3465015808.000000, d: 2.172514e-311)
fpr12 000000003424b220 (f: 874820096.000000, d: 4.322186e-315)
fpr13 000003ffd087d688 (f: 3498563328.000000, d: 2.172530e-311)
fpr14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
fpr15 000003ffc747ea28 (f: 3343378944.000000, d: 2.172454e-311)
Module=/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_s390x_linux_Nightly/openjdkbinary/j2sdk-image/lib/compressedrefs/libj9jit29.so
Module_base_address=000003FF9B300000
Target=2_90_20200118_267 (Linux 4.4.0-170-generic)
CPU=s390x (4 logical CPUs) (0x1f723a000 RAM)

JavaVMInitArgs.nOptions=19:
    -Xoptionsfile=/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_s390x_linux_Nightly/openjdkbinary/j2sdk-image/lib/options.default
    -Xlockword:mode=default,noLockword=java/lang/String,noLockword=java/util/MapEntry,noLockword=java/util/HashMap$Entry,noLockword=org/apache/harmony/luni/util/ModifiedMap$Entry,noLockword=java/util/Hashtable$Entry,noLockword=java/lang/invoke/MethodType,noLockword=java/lang/invoke/MethodHandle,noLockword=java/lang/invoke/CollectHandle,noLockword=java/lang/invoke/ConstructorHandle,noLockword=java/lang/invoke/ConvertHandle,noLockword=java/lang/invoke/ArgumentConversionHandle,noLockword=java/lang/invoke/AsTypeHandle,noLockword=java/lang/invoke/ExplicitCastHandle,noLockword=java/lang/invoke/FilterReturnHandle,noLockword=java/lang/invoke/DirectHandle,noLockword=java/lang/invoke/ReceiverBoundHandle,noLockword=java/lang/invoke/DynamicInvokerHandle,noLockword=java/lang/invoke/FieldHandle,noLockword=java/lang/invoke/FieldGetterHandle,noLockword=java/lang/invoke/FieldSetterHandle,noLockword=java/lang/invoke/StaticFieldGetterHandle,noLockword=java/lang/invoke/StaticFieldSetterHandle,noLockword=java/lang/invoke/IndirectHandle,noLockword=java/lang/invoke/InterfaceHandle,noLockword=java/lang/invoke/VirtualHandle,noLockword=java/lang/invoke/PrimitiveHandle,noLockword=java/lang/invoke/InvokeExactHandle,noLockword=java/lang/invoke/InvokeGenericHandle,noLockword=java/lang/invoke/VarargsCollectorHandle,noLockword=java/lang/invoke/ThunkTuple
    -Xjcl:jclse29
    -Dcom.ibm.oti.vm.bootstrap.library.path=/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_s390x_linux_Nightly/openjdkbinary/j2sdk-image/lib/compressedrefs:/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_s390x_linux_Nightly/| 8.03s  | 70.0MB/s  | 54.8MB/s  | 112MB      | 440MB   | 1.85GB  |
| 9.03s  | 70.0MB/s  | 54.5MB/s  | 108MB      | 492MB   | 1.79GB  |
| 10.0s  | 70.0MB/s  | 54.4MB/s  | 126MB      | 545MB   | 1.73GB  |
| 11.0s  | 70.0MB/s  | 55.5MB/s  | 123MB      | 612MB   | 1.65GB  |
JVMPORT030W /proc/sys/kernel/core_pattern setting "|/usr/share/apport/apport %p %s %c %d %P" specifies that the core dump is to be piped to an external program.  Attempting to rename either core or core.26083.


Generated system dump: core
----------- Stack Backtrace -----------
(0x000003FF9B492DD4 [libj9jit29.so+0x192dd4])
(0x000003FF9B49309C [libj9jit29.so+0x19309c])
(0x000003FF9B4A0E0E [libj9jit29.so+0x1a0e0e])
(0x000003FF9B4AB396 [libj9jit29.so+0x1ab396])
(0x000003FF9B4AE29C [libj9jit29.so+0x1ae29c])
(0x000003FF9B47B9D6 [libj9jit29.so+0x17b9d6])
(0x000003FFA0D81570 [libj9hookable29.so+0x1570])
(0x000003FFA0F37DC4 [libj9vm29.so+0xb7dc4])
(0x000003FFA0F3907A [libj9vm29.so+0xb907a])
(0x000003FFA0F3CB6C [libj9vm29.so+0xbcb6c])
(0x000003FFA0F31C76 [libj9vm29.so+0xb1c76])
(0x000003FFA0EB942A [libj9vm29.so+0x3942a])
(0x000003FFA0E9421E [libj9vm29.so+0x1421e])
(0x000003FFA0F88AC4 [libj9vm29.so+0x108ac4])
---------------------------------------

SyntheticGCWorkload_DoubleMap_J9_0_FAILED
```

**Comment by pshipton** [comment]
This was already resolved, see #8339.
